variables.tf

```diff
@@ -47,16 +47,16 @@ variable "provisionaccount_role_name" {
   default     = "ProvisionAccount"
 }
 
-variable "provisionvenom_policy_description" {
+variable "provisioncdm_policy_description" {
   type        = string
-  description = "The description to associate with the IAM policy that allows provisioning of the VENOM layer in the Shared Services account."
-  default     = "Allows provisioning of the VENOM layer in the Shared Services account."
+  description = "The description to associate with the IAM policy that allows provisioning of the CDM layer in the Shared Services account."
+  default     = "Allows provisioning of the CDM layer in the Shared Services account."
 }
 
-variable "provisionvenom_policy_name" {
+variable "provisioncdm_policy_name" {
   type        = string
-  description = "The name to assign the IAM policy that allows provisioning of the VENOM layer in the Shared Services account."
-  default     = "ProvisionVenom"
+  description = "The name to assign the IAM policy that allows provisioning of the CDM layer in the Shared Services account."
+  default     = "ProvisionCdm"
```

Comment:
Are we opposed to `ProvisionCDM` here?

Reply:
We didn't use `ProvisionVE**M` before.  In general I default to camel-casing acronyms when camel casing variable names, since it maximizes readability IMO.  (Not saying it has to be that way, just that it's what I do.)

Reply:
I get it.  When I see a camel-cased acronym, it just looks odd to me.  Not a huge deal either way.